netbox_branching/jobs.py

```diff
@@ -122,6 +134,12 @@ class RevertBranchJob(JobRunner):
     """
     class Meta:
         name = 'Revert branch'
+        job_timeout = 3600  # 1 hour - increased for large reverts
+
+    @property
+    def job_timeout(self):
+        """Return the job timeout from Meta."""
+        return getattr(self.Meta, 'job_timeout', None)
```

Comment:
This seems a bit redundant. Any way to factor it out to a single implementation?